gateway/src/wiFiHandler.cpp

```diff
@@ -0,0 +1,87 @@
+#include "wiFiHandler.h"
+
+#include "ArduinoJson.h"
+#include "HTTPClient.h"
+#include "data.pb.h"
+
+const char* ssid = "YOUR_SSID";
+const char* password = "YOUR_PASSWORD";
+const char* sensorCommunityUrl = "http://api.sensor.community/v1/push-sensor-data/";
+
+// HTTP request to sensor.community
+void requestSensorCommunity(String xSensor, String xPin) {
+  if (WiFi.status() == WL_CONNECTED) {
+    HTTPClient http;
+
+    // HTTP request construction with headers
+    http.begin(sensorCommunityUrl);
+    http.addHeader("Content-Type", "application/json");
+    http.addHeader("X-Sensor", xSensor);
+    http.addHeader("X-Pin", xPin);
+
+    // JSON construction
+    JsonDocument doc;
+    JsonArray sensordatavalues = doc["sensordatavalues"].to<JsonArray>();
+    // Add object on array
+    JsonObject value0 = sensordatavalues.createNestedObject();
```

Comment:
You can strongly type with `add<T>`


```suggestion
    JsonObject value0 = sensordatavalues.add<JsonObject>();
```